app/services/analytics_events.rb

```diff
@@ -530,6 +530,52 @@ def idv_personal_key_confirm_visited(*)
     track_event('IdV: personal key confirm visited')
   end
 
+  # @identity.idp.event_name IdV: personal key submitted
+  # The user submitted their personal key on the personal key page
+  def idv_personal_key_submitted
+    track_event('IdV: personal key submitted')
+  end
+
+  # @identity.idp.event_name IdV: personal key visited
+  # The user visited the personal key page
+  def idv_personal_key_visited
+    track_event('IdV: personal key visited')
+  end
+
```

Comment:
looks like these two are already defined in #6285, so we can remove them from here?
```suggestion
```